packages/shared/src/utils/constants/contract.ts

```diff
@@ -619,3 +619,19 @@ export const isCallPut = (trade_type: 'rise_fall' | 'rise_fall_equal' | 'high_lo
     trade_type === TRADE_TYPES.RISE_FALL ||
     trade_type === TRADE_TYPES.RISE_FALL_EQUAL ||
     trade_type === TRADE_TYPES.HIGH_LOW;
+
+export const getContractBasis = (contract_type = '') => {
```

Comment:
we are using 'basis' term for stake and payout, it's better to rename.
Actually, another idea's just hit me 💡: I'd like to suggest updating the `getSupportedContracts` config above where we added `button_name` long time ago when we faced a similar situation.
Can we please try adding an additional field like 'main_title' to `getSupportedContracts` for Turbos, Vanillas and Multipliers in order to use it instead of this new helper? 🙏🏻

Reply:
Will try

Reply:
I added `main_title` field to the contract config and expand the functionality of `getTradeTypeName` function (which call `getSupportedContracts` deep inside)